pkg/virt-launcher/monitor_test.go

```diff
@@ -209,13 +211,17 @@ var _ = Describe("VirtLauncher", func() {
 				go func() { CleanupProcess() }()
 
 				go func() {
+					defer GinkgoRecover()
 					mon.monitorLoop(1*time.Second, stopChan)
 					done <- "exit"
 				}()
 
 				Consistently(gracefulShutdownChannel).Should(Not(BeClosed()))
 				close(stopChan)
 				Eventually(gracefulShutdownChannel).Should(BeClosed())
+
+				StopProcess()
```

Comment:
It looks like we StopProcess in the AfterEach, why do we need to do it in the test too?

Reply:
Hm...Trying to remember what was the idea behind this ~12 days ago :) Now after having one more fresh look I think that indeed it does not seem right. The issue is hard to reporduce and perhaps this change just affected the probability of failure... I may need to investigate it further. Thanks for poinoint on that.